### DISCUSSION TITLE
Anyone test this on K2 Plus yet?

### DISCUSSION THREAD

**Discussion by Ltek** [Guilouz/Creality-Helper-Script-Wiki#688] [General]
Curious if this has been tested on K2 Plus yet... would be great if this works on it.

**Comment by GuardianArias** [comment]
Second this, I ordered a K2 because creality's support provided me with the root username and password already. 

BTW they are:

Account: root
Password: creality_2024

**Comment by asgoldenberg** [comment]
If you have a K2, what are your hesitations to trying these tools? If I were to guess, maybe there is a scenario you are concerned your changes will be irreversible and that you end up with a bricked K2?

**Comment by Guilouz** [comment]
Currently the script is not compatible with K2, the OS is slightly different.

I'm waiting for Creality to send it to me. Normally in December.
